pkg/asset/installconfig/azure/client.go

```diff
@@ -439,3 +443,30 @@ func (c *Client) GetLocationInfo(ctx context.Context, region string, instanceTyp
 
 	return nil, fmt.Errorf("location information not found for %s in %s", instanceType, region)
 }
+
+// CheckStorageAccount checks if the storage account URI provided already exists for diagnostics
+// purposes.
+func (c *Client) CheckStorageAccount(ctx context.Context, storageAccountURI string) error {
```

Comment:
Maybe be more specific about what we're checking--confirming the existence of the storage account.